docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -771,9 +770,9 @@ The output of the query has the following columns:
 ``$all_files`` table
 ^^^^^^^^^^^^^^^^^^^^
 
-The ``$all_files`` table provides a detailed overview of the data file in all
+The ``$all_files`` table provides a detailed overview of the data files in all
```

Comment:
a minor issue with commit boundary, can you please pull out the unrelated refactors in separate commit? https://github.com/trinodb/trino/pull/11206#pullrequestreview-936713776

Reply:
@phd3 thanks, I'll take cream of these comments latter this week.
Got busy into some other work.